steps/jira-tickets/0.0.1/step.yml

```diff
@@ -0,0 +1,71 @@
+title: JIRA Tickets listing
+summary: |
+  List all JIRA tickets from comparing two branches
+description: |
+  List all JIRA tickets from comparing two branches
```

Comment:
It does not make sense to have the same summary and description.
Try reformulating them.